docker/gaffer-integration-tests/docker-compose.yaml

```diff
@@ -0,0 +1,177 @@
+    # file structure it needs. Using the following workaround to allow
+    # accumulo to "auth" with HDFS as the super user so that it can:
+    - HADOOP_USER_NAME=hadoop
+    volumes:
+    - ../accumulo/conf:${ACCUMULO_CONF_DIR}:ro
+    - /var/log/accumulo
+
+  accumulo-tserver:
+    depends_on:
+    - accumulo-master
+    image: gchq/gaffer:${GAFFER_VERSION}
+    command: tserver
+    restart: on-failure
+    container_name: accumulo-tserver
+    hostname: accumulo-tserver
+    environment:
+    - ACCUMULO_CONF_DIR=${ACCUMULO_CONF_DIR}
+    # There doesn't seem to be an easy way (with docker-compose) to init our
+    # HDFS instance with the right permissions so that Accumulo can create the
+    # file structure it needs. Using the following workaround to allow
+    # accumulo to "auth" with HDFS as the super user so that it can:
+    - HADOOP_USER_NAME=hadoop
+    volumes:
+    - ../accumulo/conf:${ACCUMULO_CONF_DIR}:ro
+    - /var/log/accumulo
+
+  accumulo-monitor:
+    depends_on:
+    - accumulo-master
+    image: gchq/gaffer:${GAFFER_VERSION}
+    command: monitor
+    restart: on-failure
+    container_name: accumulo-monitor
+    hostname: accumulo-monitor
+    environment:
+    - ACCUMULO_CONF_DIR=${ACCUMULO_CONF_DIR}
+    # There doesn't seem to be an easy way (with docker-compose) to init our
+    # HDFS instance with the right permissions so that Accumulo can create the
+    # file structure it needs. Using the following workaround to allow
+    # accumulo to "auth" with HDFS as the super user so that it can:
+    - HADOOP_USER_NAME=hadoop
+    ports:
+    - 9995:9995
+    volumes:
+    - ../accumulo/conf:${ACCUMULO_CONF_DIR}:ro
+    - /var/log/accumulo
+
+  accumulo-gc:
+    depends_on:
+    - accumulo-master
+    image: gchq/gaffer:${GAFFER_VERSION}
+    command: gc
+    restart: on-failure
+    container_name: accumulo-gc
+    hostname: accumulo-gc
+    environment:
+    - ACCUMULO_CONF_DIR=${ACCUMULO_CONF_DIR}
+    # There doesn't seem to be an easy way (with docker-compose) to init our
+    # HDFS instance with the right permissions so that Accumulo can create the
+    # file structure it needs. Using the following workaround to allow
+    # accumulo to "auth" with HDFS as the super user so that it can:
+    - HADOOP_USER_NAME=hadoop
+    volumes:
+    - ../accumulo/conf:${ACCUMULO_CONF_DIR}:ro
+    - /var/log/accumulo
+
+  gaffer-rest:
+    image: gchq/gaffer-rest:${GAFFER_VERSION}
+    build:
+      context: ../gaffer-rest/
+      args:
+        GAFFER_VERSION: ${GAFFER_VERSION}
+        GAFFER_TOOLS_VERSION: ${GAFFER_TOOLS_VERSION}
+    ports:
+    - 8080:8080
+    volumes:
+    - ../gaffer/conf/store.properties:/gaffer/store/store.properties:ro
+    - ./config/graph/graphConfig.json:/gaffer/graph/graphConfig.json:ro
+    - ./config/schema:/gaffer/schema:ro
+    - ./config/ui/config.json:/opt/jboss/wildfly/standalone/deployments/ui.war/config/config.json:ro
+
+  data-loader:
```

Comment:
For the integration tests you might not need the data loader as they tend to blat the accumulo table before starting